src/libraries/System.Private.CoreLib/src/System/Half.cs

```diff
@@ -385,27 +385,28 @@ public int CompareTo(object? obj)
         /// <returns>A value less than zero if this is less than <paramref name="other"/>, zero if this is equal to <paramref name="other"/>, or a value greater than zero if this is greater than <paramref name="other"/>.</returns>
         public int CompareTo(Half other)
```

Comment:
Could you share the before/after disassembly and possible some perf numbers showing this is an improvement?

Reply:
Yup, I'd like to see this too. I'm curious about how much improvement we'd really see here.